cfme/infrastructure/resource_pool.py

```diff
@@ -151,7 +150,7 @@ def delete(self, cancel=False, wait=False):
             def refresh():
                 if self.provider:
                     self.provider.refresh_provider_relationships()
-                view.browser.selenium.refresh()
+                view.browser.refresh()
```

Comment:
**QUESTION**
What's the point of this change?

Reply:
both work same to hide `selenium` @quarckster